R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -1,147 +1,322 @@
 #' @export
-setClass(
-  "TableHandle",
-  representation(
-    .internal_rcpp_object = "Rcpp_INTERNAL_TableHandle"
-  )
-)
+TableHandle <- R6Class("TableHandle",
+  cloneable = FALSE,
+  public = list(
+    
+    .internal_rcpp_object = NULL,
+    
+    initialize = function(table_handle) {
+      if (class(table_handle)[[1]] != "Rcpp_INTERNAL_TableHandle") {
+        stop("'table_handle' should be an internal Deephaven TableHandle. If you're seeing this,
+                you are trying to call the constructor of TableHandle directly, which is not advised.")
+      }
+      self$.internal_rcpp_object <- table_handle
+    },
 
-### TABLEHANDLE PROPERTIES
+    #' @description
+    #' Whether the table referenced by this TableHandle is static or not.
+    #' @return TRUE if the table is static, or FALSE if the table is ticking.
+    is_static = function() {
+      return(self$.internal_rcpp_object$is_static())
+    },
+    
+    #' @description
+    #' Binds the table referenced by this TableHandle to a variable on the server,
+    #' enabling it to be accessed by that name from any Deephaven API.
+    #' @param name Name for this table on the server.
```

Comment:
I'm not reviewing the doc strings, as you suggested.  I would remove them so that they all actually get reviewed in the docs pr.